src/Sylius/Bundle/ApiBundle/Payment/ApiPaymentMethodHandlerInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Payment;
+
+use Sylius\Component\Core\Model\PaymentInterface;
+use Sylius\Component\Core\Model\PaymentMethodInterface;
+
+/** @experimental */
+interface ApiPaymentMethodHandlerInterface
```

Comment:
I wonder if the better name would not be:
```suggestion
interface ApiPaymentConfigurationProviderInterface
```
or sth like that, but as it is an experimental feature, it could be refactored later

Reply:
IMHO we shouldn't leave not perfect names because it is `the experimental feature`, we can leave it as it is if this name is good enough to merge :D

Reply:
The service is already in ApiBundle and therefore, API prefix for this interface seems for me redundant. Also, these providers do not return anything API based. It provides an array of configurations, which will be used in API context later. Service itself is not coupled to API itself
```suggestion
interface PaymentConfigurationProviderInterface
```